### PR TITLE
feat(jsonschemax): Add readOnly flag to path

### PR DESCRIPTION
**Pull request by zepatrik**

### CODE REVIEW

File: jsonschemax/keys_test.go [hunk 1]
```diff
@@ -304,7 +304,8 @@ func TestListPaths(t *testing.T) {
         },
         "bar": {
           "type": "boolean",
-          "default": "asdf"
+          "default": "asdf",
+		  "readOnly": true
```
Comment (aeneasr) [inline jsonschemax/keys_test.go]: Please run `make format`

**Comment by aeneasr** [comment]
Merge conflict ;)

**Comment by aeneasr** [comment]
ping @zepatrik

**Comment by zepatrik** [comment]
done @aeneasr

**Pull request merged**